packages/components/Badge/src/Badge.types.ts

```diff
@@ -170,10 +203,16 @@ export interface BadgeTokens extends BadgeCoreTokens, BadgeConfigurableProps {
   subtle?: BadgeTokens;
   success?: BadgeTokens;
   warning?: BadgeTokens;
+  disabled?: BadgeTokens;
+
+  /**
+   * Selected state tokens for Badge
+   */
+  selected?: BadgeTokens;
 }
 
 export interface BadgeSlotProps {
-  root: IViewProps;
+  root: React.PropsWithRef<PressablePropsExtended>;
```

Comment:
do you need slot with Ref here & above ?

Reply:
Removed